deepchem/utils/vina_utils.py

```diff
@@ -56,84 +78,7 @@ def write_vina_conf(protein_filename: str,
       f.write("exhaustiveness = %d\n" % exhaustiveness)
 
 
-def write_gnina_conf(protein_filename: str,
-                     ligand_filename: str,
-                     conf_filename: str,
-                     num_modes: int = 9,
-                     exhaustiveness: int = None,
-                     **kwargs) -> None:
-  """Writes GNINA configuration file to disk.
-
-  GNINA accepts a configuration file which provides options
-  under which GNINA is invoked. This utility function writes a
-  configuration file which directs GNINA to perform docking
-  under the provided options.
-
-  Parameters
-  ----------
-  protein_filename: str
-    Filename for protein
-  ligand_filename: str
-    Filename for the ligand
-  conf_filename: str
-    Filename to write Autodock Vina configuration to.
-  num_modes: int, optional (default 9)
-    The number of binding modes GNINA should find
-  exhaustiveness: int, optional
-    The exhaustiveness of the search to be performed by GNINA
-  kwargs:
-    Args supported by GNINA documented here
-    https://github.com/gnina/gnina#usage
-
-  """
-
-  with open(conf_filename, "w") as f:
-    f.write("receptor = %s\n" % protein_filename)
-    f.write("ligand = %s\n\n" % ligand_filename)
-
-    f.write("autobox_ligand = %s\n\n" % protein_filename)
-
-    if exhaustiveness is not None:
-      f.write("exhaustiveness = %d\n" % exhaustiveness)
-    f.write("num_modes = %d\n\n" % num_modes)
-
-    for k, v in kwargs.items():
-      f.write("%s = %s\n" % (str(k), str(v)))
-
-
-def read_gnina_log(log_file: str) -> np.array:
-  """Read GNINA logfile and get docking scores.
-
-  GNINA writes computed binding affinities to a logfile.
-
-  Parameters
-  ----------
-  log_file: str
-    Filename of logfile generated by GNINA.
-
-  Returns
-  -------
-  scores: np.array, dimension (num_modes, 3)
-    Array of binding affinity (kcal/mol), CNN pose score,
-    and CNN affinity for each binding mode.
-
-  """
-
-  scores = []
-  lines = open(log_file).readlines()
-  mode_start = np.inf
-  for idx, line in enumerate(lines):
-    if line[:6] == '-----+':
-      mode_start = idx
-    if idx > mode_start:
-      mode = line.split()
-      score = [float(x) for x in mode[1:]]
-      scores.append(score)
-
-  scores = np.array(scores)
-  return scores
-
-
+@deprecated
 def load_docked_ligands(
```

Comment:
Perhaps we could import the functions from `docking_utils.py` and then have only deprecation wrappers in `vina_utils.py` so we don't have duplicate code?

Reply:
Got it, will do!